tests/test_extensions.py

```diff
@@ -1121,14 +1121,35 @@ def testBacklinkText(self):
         text = 'paragraph[^1]\n\n[^1]: A Footnote'
         self.assertEqual(
             md.convert(text),
-            '<p>paragraph<sup id="fnref:1"><a class="footnote-ref" href="#fn:1" rel="footnote">1</a></sup></p>\n'
+            '<p>paragraph<sup id="fnref:1"><a class="footnote-ref" href="#fn:1">1</a></sup></p>\n'
             '<div class="footnote">\n'
             '<hr />\n'
             '<ol>\n'
             '<li id="fn:1">\n'
-            '<p>A Footnote&#160;<a class="footnote-backref" href="#fnref:1" rev="footnote"'
+            '<p>A Footnote&#160;<a class="footnote-backref" href="#fnref:1"'
             ' title="Jump back to footnote 1 in the text">back</a></p>\n'
             '</li>\n'
             '</ol>\n'
             '</div>'
         )
+
+    def testFootnoteSeparator(self):
```

Comment:
Let's add this new test using the new testing framework. Ideally the existing tests should be moved as well. But I'll settle for the new one only for now.

Reply:
I'll most likely move all footnote tests in this file to the new format.